plugins/python_3_script/requirements.txt

```diff
@@ -5,7 +5,8 @@ records==0.5.3
 maya==0.6.1
 lxml==4.9.2
 datetime==5.1
-setuptools==65.5.1
+setuptools==70.0.0
 PyYAML==6.0.1
 beautifulsoup4==4.12.2
 parameterized==0.8.1
+zipp==3.19.1
```

Comment:
Should we replace pathlib imports with zipp in the run action?

Reply:
Both maya and records introduce the vulnerability. Pathlib didn't contain any snyk vulns